docker/sqlg/gremlin-server.yaml

```diff
@@ -1,5 +1,6 @@
 host: localhost
 port: 8182
+evaluationTimeout: 0
```

Comment:
Could we add a comment here about how setting this to 0 disables the timeout entirely?

Reply:
```suggestion
# Time in milliseconds that an evaluation is allowed to run and its results potentially transformed. Set to zero to have no timeout set.
evaluationTimeout: 0
```